Calculator/Calculator/Controller/CalculatorViewController.swift

```diff
@@ -14,149 +14,175 @@ class CalculatorViewController: UIViewController {
         }
     }
     
     @IBAction private func dotButtonDidTap(_ sender: UIButton) {
-        if operandNumber.isEmpty {
-            operandNumber = CalculatorValue.zeroDecimal
-            operandLabel.text = numberFormatter(operand: operandNumber)
-        } else if operandNumber.last == Character(CalculatorValue.decimalPoint) || operandNumber.count == 20 || operandNumber.contains(Character(CalculatorValue.decimalPoint)) {
-            operandLabel.text = numberFormatter(operand: operandNumber)
+        guard currentOperand.count < 20 &&
+                !currentOperand.contains(Character(CalculatorValue.decimalPoint)) else { return }
+   
+        if currentOperand.isEmpty || currentOperand == CalculatorValue.minus {
+            currentOperand += CalculatorValue.zeroDecimal
         } else {
-            operandNumber += CalculatorValue.decimalPoint
-            operandLabel.text = operandNumber
+            currentOperand += CalculatorValue.decimalPoint
         }
+        
+        operandLabel.text = currentOperand
     }
     
-    @IBAction private func operatorButtonDidTap(_ sender: UIButton) {
-        guard let operatorSymbol = sender.title(for: .normal) else { return }
+    private func addListStackView() -> UIStackView {
+        let operatorLabel = UILabel()
+        let operandLabel = UILabel()
+        let stackView = UIStackView()
         
-        if operandLabel.text == CalculatorValue.zero {
-            operatorLabel.text = operatorSymbol
+        stackView.spacing = 8
+        operandLabel.textColor = .white
+        operatorLabel.textColor = .white
+        operandLabel.font = UIFont.preferredFont(forTextStyle: .title3)
+        operatorLabel.font = UIFont.preferredFont(forTextStyle: .title3)
+        
+        if currentOperand == CalculatorValue.minus {
+            operandLabel.text = currentOperand + CalculatorValue.zero
         } else {
-            formulaNumber += operatorLabel.text ?? CalculatorValue.emptyArray
-            formulaNumber += operandLabel.text ?? CalculatorValue.emptyArray
-            addScrollStackView()
-            operandNumber = CalculatorValue.emptyArray
-            operandLabel.text = CalculatorValue.zero
-            operatorLabel.text = operatorSymbol
+            operandLabel.text = currentOperand
         }
+        operatorLabel.text = currentOperator
+        
+        stackView.addArrangedSubview(operatorLabel)
+        stackView.addArrangedSubview(operandLabel)
+        
+        return stackView
     }
     
-    @IBAction private func allClearButtonDidTap(_ sender: Any) {
-        initializeUI()
-        formulaNumber = CalculatorValue.emptyArray
-        operandNumber = CalculatorValue.emptyArray
+    private func scrollToBottom() {
+        formulaScrollView.layoutIfNeeded()
+        formulaScrollView.setContentOffset(CGPoint(x: 0, y: formulaScrollView.contentSize.height - formulaScrollView.bounds.height), animated: false)
```

Comment:
메서드 호출 코드가 너무 길 때는 파라미터마다 개행을 해주기도 합니다.
참고만 해주세요!
```suggestion
        formulaScrollView.setContentOffset(
            CGPoint(x: 0, y: formulaScrollView.contentSize.height - formulaScrollView.bounds.height),
            animated: false
        )
```